packages/wonder-blocks-form/src/components/text-field.tsx

```diff
@@ -247,12 +249,15 @@ class TextField extends React.Component<PropsWithForwardRef, State> {
                         ? styles.disabled
                         : this.state.focused
                         ? [styles.focused, light && styles.defaultLight]
-                        : this.state.error && [
+                        : !!this.state.error && [
                               styles.error,
                               light && styles.errorLight,
                           ],
                     style && style,
```

Comment:
**suggestion:** If a component has a `style` prop, it's usually placed last so that it can override the default styles.  Can you move this to be last?

Reply:
Sorry, I didn't explain myself well.  The item `style && style,` within the array we pass to `style` should come last.  The order of the props passed to the `StyledInput` component don't matter because `<StyledInput id={id} type={type} style={[...]} />` gets turned into:
```
React.createElement(StyledInput, {
   id: id,
   type: type,
   style: [ ... ],
});
```
The only time order of props matters is when prop spreading is involved, e.g.
```
const props = {message: "hello", name: "world"};
<MyComponent {...props} message="bonjour" />; // "bonjour" will override props.message
<MyComponent message="bonjour" {...props} />; // props.message will override "bonjour"
```